### PR TITLE
feat(website): add gourds to ecosystem

### LINKED ISSUE #14809: Gourds
### Title of your project

Gourds

### Short (1-2 sentence) description of your project

Gourds aims to achieve crypto market efficiency by introducing novel trading instruments for mass adoption

### Link to your project

https://gourds.studio

### Live on Testnet

Yes (default)

### If your project is live on one of Taiko's testnets, please provide evidence of that here (eg. links, screenshots). If your project is NOT live, please provide your plan for integration here.

https://gourds.studio

### Upload a quality logo to your project

logo1

### PR DESCRIPTION
**Pull request by 2manslkh**
Closes #14809

### CODE REVIEW

**Review by davidtaikocha — DISMISSED** [review]

**Comment by MarcusWentz** [comment]
I see Gourds mentioned twice.

**Review by MarcusWentz — CHANGES_REQUESTED** [review]
I see Gourds mentioned twice.

**Review by dionysuzx — DISMISSED** [review]

File: packages/website/components/Ecosystem/EcosystemSection.tsx [hunk 1]
```diff
@@ -330,6 +339,15 @@ const ecosystemData: EcosystemData[] = [
     filters: [],
     isLive: true,
   },
+  {
+    icon: "/images/ecosystem/gourds.jpeg",
+    name: "Gourds",
```
Comment (dantaik) [inline packages/website/components/Ecosystem/EcosystemSection.tsx:344]: Duplicate

File: packages/website/components/Ecosystem/EcosystemSection.tsx [hunk 2]
```diff
@@ -330,6 +339,15 @@ const ecosystemData: EcosystemData[] = [
...
+    link: "https://gourds.studio",
+    description:
+      "Gourds aims to achieve crypto market efficiency by introducing novel trading instruments for mass adoption.",
+    filters: [],
+    isLive: true,
+  },
```
Comment (d1onys1us) [inline packages/website/components/Ecosystem/EcosystemSection.tsx:350]:
```suggestion
```

**Review by dionysuzx — APPROVED** [review]

**Review by MarcusWentz — APPROVED** [review]

**Pull request merged**